tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -129,8 +129,16 @@ def validate(self, ctx):
         if self.topic is None:
             return False
         ctx.redpanda.logger.info(f"Validating topic {self.topic} deletion")
-        topics = ctx.rpk().list_topics()
-        return self.topic not in topics
+
+        all_topics = set()
+        # since metadata in Redpanda and Kafka are eventually consistent
+        # we must check all the nodes before proceeding
+        for n in ctx.redpanda.started_nodes():
+            partitions = ctx.admin().get_partitions(node=n)
+            for p in partitions:
+                all_topics.add(p['topic'])
+
+        return self.topic not in all_topics
```

Comment:
Let's use the following snippet, it maps directly to the comment and does logging what is super useful when debugging ci-failures

```
for n in ctx.redpanda.started_nodes():
    partitions = ctx.admin().get_partitions(node=n)
    for p in partitions:
      if p['topic'] == self.topic:
        ctx.redpanda.logger.info(f"found topic {self.topic} on node {n.account.hostname}")
        return False

return True
```